app/src/org/commcare/activities/EntitySelectActivity.java

```diff
@@ -173,6 +174,9 @@ protected void onCreate(Bundle savedInstanceState) {
         refreshTimer = new EntitySelectRefreshTimer();
         asw = CommCareApplication._().getCurrentSessionWrapper();
         session = asw.getSession();
+        // Don't show actions (e.g. 'register patient', 'claim patient') when
+        // in the middle on workflow triggered by an (sync) action.
+        hideActions = session.isSyncCommand(session.getCommand());

```

Comment:
(Just making a note of the fact that we decided to wait to address this until after the first round of UI/UX feedback)